packages/environment-check/src/checks/install.ts

```diff
@@ -0,0 +1,64 @@
+import { existsSync, promises } from 'fs';
+import { t } from '../i18n';
+import { join } from 'path';
+import { spawnCommand, npmCommand } from '../command';
+
+/**
+ * Reads the list of extensions installed in vscode.
+ *
+ * @returns list of extension ids
+ */
+export const getInstalledExtensions = async (): Promise<{ [id: string]: { version: string } }> => {
+    const output = await spawnCommand('code', ['--list-extensions', '--show-versions']);
+    const versions = output
+        .split('\n')
+        .filter((ext) => ext.startsWith('SAP'))
+        .reduce((returnObject, current) => {
+            const [id, version] = current.split('@');
+            returnObject[id] = {
+                version
+            };
+            return returnObject;
+        }, {});
+    return versions;
+};
+
+/**
+ * Read the version of the cloud foundry CLI.
+ *
+ * @param module - module id
+ * @returns version
+ */
+export const getCFCliToolVersion = async (module: string): Promise<string> => {
+    let cfVersion: string;
+    try {
+        const ouput = await spawnCommand(module, ['-v']);
```

Comment:
```suggestion
        const ouput = await spawnCommand(NpmModules.CloudCliTools, ['-v']);
```
Requires also to add 
```
import { NpmModules } from '../types';
```
to the import section. 

Also there is a type: ouput instead of output

Reply:
Updated as suggested